internal/core/application/operator_service.go

```diff
@@ -171,43 +168,53 @@ func (o *operatorService) DepositMarket(
 		}
 
 		nextAccountIndex := latestAccountIndex + 1
-		fee := o.marketFee
```

Comment:
can you explain this deletion?

Reply:
o.marketFee is used only when invoking GetOrCreateMarket... so I found excessive declaring a variable for that and I removed it... now o.marketFee is directly used as arg of GetOrCreateMarket invocation

Reply:
I meant the whole block of code

Reply:
Since this RPC makes cross repo changes (to vault and possible market repos), all the write operations are required to be executed within a non-managed db tx to enforce strong consistency. For this reason, the RPC now sources the domains or create new ones, and after it's done its job, applies the changes to the repo in a tx managed by the repo manager..